addons/website/views/website_templates.xml

```diff
@@ -2231,7 +2231,7 @@
             <div class="container">
                 <div class="alert alert-info text-center d-lg-flex justify-content-between align-items-center">
                     <p class="m-lg-0 text-info">This page does not exist, but you can create it as you are editor of this site.</p>
-                    <a role="button" class="btn btn-info js_disable_on_click post_link" t-attf-href="/website/add/#{path}?redirect=1#{from_template and '&amp;template=%s' % from_template}">Create Page</a>
+                    <a role="button" class="btn btn-info js_disable_on_click post_link" data-post_force-go-to-frontend="true" t-attf-href="/website/add/#{path}?redirect=1#{from_template and '&amp;template=%s' % from_template}">Create Page</a>
```

Comment:
isn't it `force-go-to-backend` and not `frontend`?
I think I know what you mean but it actually about forcing to reload top window and not the iframe (it's what it does, adding `_top` as target, shouldn't it just be named accordingly to avoid confusion?
`data-post_force-top-window`